snippets/standard/datetime/json/custom-reading-with-utf8jsonreader.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Globalization;
+using System.Text;
+using System.Text.Json;
+
+public class Example
+{
+    public static void Main(string[] args)
+    {
+        byte[] utf8Data = Encoding.UTF8.GetBytes(@"""Friday, 26 July 2019 00:00:00""");
+
+        Utf8JsonReader json = new Utf8JsonReader(utf8Data);
+        while (json.Read())
+        {
+            if (json.TokenType == JsonTokenType.String)
+            {
+                string value = json.GetString();
```

Comment:
I think it would be good to include what happens if you just call `json.GetDateTime()`. Documenting the exception and behavior before showing the work-around would make things clearer.